### PR TITLE
[API-41134] Add POA requests docs

### PR DESCRIPTION
**Pull request by tycol7**
## Summary

- Update POA requests docs (staging only).
- Fix snake_case response (to camelCase) in `read_poa_request`.
- Fix import error for `BadGateway` error.

## Related issue(s)

[API-41134](https://jira.devops.va.gov/browse/API-41134)

## Testing done

- [x] *New code is covered by unit tests*

## Screenshots

poa_requests_docs

## What areas of the site does it impact?

- POA requests docs (staging only)
- POA request controller
- ManageRepresentativeService `read_poa_request`

## Acceptance criteria

- [x]  I updated unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature

## Requested Feedback

N/A

### CODE REVIEW

File: modules/claims_api/config/schemas/v2/power_of_attorney_requests/param/decision/post.json [hunk 1]
```diff
@@ -10,7 +10,6 @@
       "type": "object",
       "additionalProperties": false,
       "required": [
-        "type",
```
Comment (tycol7) [inline modules/claims_api/config/schemas/v2/power_of_attorney_requests/param/decision/post.json:13]: Not related to this task, but realized we aren't using this `type` in the decision docs.

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json [hunk 2]
```diff
@@ -407,659 +406,470 @@
...
-                      "page": {
-                        "number": 2,
-                        "size": 3
-                      },
-                      "sort": {
-                        "field": "createdAt",
-                        "order": "asc"
-                      }
-                    }
+                "example": [
```
Comment (FonzMP) [inline modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:418]: thinking I'd trim the cassette that was used to build this. 16 decisions / 300+ lines of code for the example is a good bit. I think we could get away with 2 or 3 examples, but 🤷

**Review by FonzMP — APPROVED** [review]
:shipit:

**Review by rmtolmach — DISMISSED** [review]

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json (hunk 2 above)
Comment (tycol7) [reply modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:418]: https://github.com/department-of-veterans-affairs/vets-api/pull/19048/commits/5fc900c7fdeb4cc72482195f2a5f707dfa7627bc

**Review by rockwellwindsor-va — APPROVED** [review]
LGTM!

**Pull request merged**